server/events.lua

```diff
@@ -40,6 +40,7 @@ AddEventHandler('playerDropped', function(reason)
     if not QBX.Players[src] then return end
     GlobalState.PlayerCount -= 1
     local player = QBX.Players[src]
+    player.PlayerData.lastLoggedOut = os.time() * 1000
```

Comment:
I don't think you should need to multiply here. I assume you are trying to add the milliseconds, but is that needed for os.date?